open-metadata-implementation/access-services/asset-manager/asset-manager-server/src/main/java/org/odpi/openmetadata/accessservices/assetmanager/handlers/SchemaExchangeHandler.java

```diff
@@ -1461,11 +1462,14 @@ public void updateSchemaAttribute(String                        userId,
                                                      getExternalSourceName(correlationProperties),
                                                      schemaAttributeGUID,
                                                      schemaAttributeGUIDParameterName,
+                                                     schemaAttributeProperties.getQualifiedName(),
+                                                     qualifiedNameParameterName,
+                                                     schemaAttributeBuilder,
+                                              null,
```

Comment:
Why is the formula called out separately in the interface since it is loaded into the builder?  If it is necessary to pull it out, why is it always null?

Reply:
I don't have access to it from the builders - so have sent through the formula from the attribute properties